Added a new stream of unit tests for rss export with scaffolding.

danfran opened the pull request:
Hi,

I started to add unit tests and I have created a scaffolding for them similar to the Snap project.

Of course more tests are necessary so I will keep on add them in case you decide to merge the current ones.

Btw I think there is a little bug with Rss 2.0, as the label for skip days is "skipDayss" with 2 final 's'.

Thanks
Daniele Francesconi

bergmark commented:
Hi Daniele,

This is very useful, thank you! I'm always happy to merge tests.

I would like to request that you add ending newlines to .gitignore and tests/Main.hs and fix the compilation warnings this introduced (it's fine to use `-fno-warn-orphans` in the test suite, please use a pragma instead of a .cabal ghc-option for it).

I released a new version fixing the `skipDayss` typo, thanks for letting me now.

Cheers

danfran commented:
I removed all the warnings I could see. Then I added missing newlines and merged your change.

Let me know if you spot other problems :)

Thanks
Daniele

bergmark commented:
Looks great, thanks!

bergmark commented:
Whoops, think I messed that up!

bergmark commented:
Double-whoops, I did not :)

bergmark commented:
oh commented in the wrong place! anyway I meant to note that these PR's were missing other-module entries, see 3ca75481b799e0568992637f761dc79dbb9fb446. It's been fixed now so no action needed.
